### DISCUSSION TITLE
User password for shopme zip file

### DISCUSSION THREAD

**Discussion by eddytjhin** [community/community#123633] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My name : eddytjhin, can i have user oqssword for zip file shopme

**Comment by LiteBrite82** [comment]
Welcome to GitHub and the GitHub Community @eddytjhin  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. 

It looks like you may have posted you question in the wrong discussion board; I was able to understand your question by looking at your profile activities. Opening an issue is one way to communicate with the owner of the repo. You may also want to try reaching out through [Udemy support](https://support.udemy.com/hc/en-us). 

For more information on how to participate in the GitHub Community, check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
